deepmd/pt/train/training.py

```diff
@@ -1030,10 +1030,13 @@ def save_model(self, save_path, lr=0.0, step=0):
             if dist.is_available() and dist.is_initialized()
             else self.wrapper
         )
-        module.train_infos["lr"] = lr
+        module.train_infos["lr"] = float(lr)
         module.train_infos["step"] = step
+        optim_state_dict = self.optimizer.state_dict()
```

Comment:
need to deep copy?

Reply:
Not necessary. `module.state_dict()` is already a copy, which will not affect the innner params.

Reply:
Here we only modified the float values, where deep copy is not necessary. Anyway for safety tensors, we should indeed use deep copy.